release/ray_release/byod/byod_rllib_test.sh

```diff
@@ -6,14 +6,15 @@ set -exo pipefail
 
 
 # TODO: https://github.com/Farama-Foundation/AutoROM/issues/48
-pip3 install https://ray-ci-deps-wheels.s3.us-west-2.amazonaws.com/AutoROM.accept_rom_license-0.5.4-py3-none-any.whl
+pip install https://ray-ci-deps-wheels.s3.us-west-2.amazonaws.com/AutoROM.accept_rom_license-0.5.4-py3-none-any.whl
```

Comment:
what is the different between `pip` and `pip3` here? `pip3` is from ubuntu base where `pip` is from conda?

Reply:
both are from conda from my testing with workspace, but they are actually two different binaries (in theory the behavior should be the same except when there are python 2 installed); but i don't want to take any chances here ;)